SECURITY.md

```diff
@@ -1,15 +1,54 @@
 # Security Policy
 
-## Supported Versions
+This document outlines the security procedures for the LoopBack project.
+
+- [Responsible Disclosure Policy](#responsible-disclosure-policy)
+  - [Reporting a Security Issue](#reporting-a-security-issue)
+- [Supported Versions](#supported-versions)
+- [Security Advisories](#security-advisories)
+
+This project participates in the Responsible Disclosure Policy program for the
+Node.js Security Ecosystem.
+
+- [Report a security vulnerability](https://hackerone.com/nodejs-ecosystem)
+
+## Responsible Disclosure Policy
+
+The LoopBack team and community take all security issues seriously. Thank you
+for improving the security of LoopBack. We appreciate your efforts and
+responsible disclosure and will make every effort to acknowledge your
+contributions.
+
+A responsible disclosure policy helps protect the project and its users from
+security vulnerabilities discovered in the project’s scope by employing a
+process where vulnerabilities are publicly disclosed after a reasonable time
+period to allow patching the vulnerability.
+
+All security bugs are taken seriously and are considered as top priority. Your
+efforts to responsibly disclose your findings are appreciated and will be taken
+into account to acknowledge your contributions.
 
-LoopBack implements the
-[Module LTS policy](https://github.com/CloudNativeJS/ModuleLTS).
+### Reporting a Security Issue
 
-| Version    | Supported          |
-| ---------- | ------------------ |
-| LoopBack 4 | :white_check_mark: |
-| LoopBack 3 | :white_check_mark: |
-| LoopBack 2 | :x:                |
+Any security related issue should be reported to the
+[Node.js Ecosystem](https://hackerone.com/nodejs-ecosystem) program hosted on
+HackerOne which follows the
+[3rd party responsible disclosure process](https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md)
+set by the Node.js Security WG.
+
+One may also directly contact the project’s maintainers by emailing
+reachsl@us.ibm.com, but through the HackerOne program the Security WG members
```

Comment:
Please use `mailto:` URL instead of a naked email.

```suggestion
[reachsl@us.ibm.com](mailto:reachsl@us.ibm.com), but through the HackerOne program the Security WG members
```